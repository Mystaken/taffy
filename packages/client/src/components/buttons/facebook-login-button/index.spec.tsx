import React from 'react';
import { create } from 'react-test-renderer';
import { FacebookLoginButton } from '.';

describe('<FacebookLoginButton />', () => {
  beforeAll(() => {
    const script = document.createElement('script');
    document.body.appendChild(script);
  });
  it('has rendered correctly', () => {
    const component = create(<FacebookLoginButton callback={() => null} />);
    const tree = component.toJSON()!;
    expect(tree).toMatchSnapshot();
  });
});
