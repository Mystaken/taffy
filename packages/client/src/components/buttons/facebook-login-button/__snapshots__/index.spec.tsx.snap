// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FacebookLoginButton /> has rendered correctly 1`] = `
<span
  style={
    Object {
      "opacity": 0.6,
      "transition": "opacity 0.5s",
    }
  }
>
  <button
    className="kep-login-facebook small"
    onClick={[Function]}
    type="button"
  >
    Login with Facebook
  </button>
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": Array [
          Array [
            9,
            ".kep-login-facebook{font-family:Helvetica,sans-serif;font-weight:700;-webkit-font-smoothing:antialiased;color:#fff;cursor:pointer;display:inline-block;font-size:calc(.27548vw + 12.71074px);text-decoration:none;text-transform:uppercase;transition:background-color .3s,border-color .3s;background-color:#4c69ba;border:calc(.06887vw + .67769px) solid #4c69ba;padding:calc(.34435vw + 13.38843px) calc(.34435vw + 18.38843px)}.kep-login-facebook.small{padding:calc(.34435vw + 3.38843px) calc(.34435vw + 8.38843px)}.kep-login-facebook.medium{padding:calc(.34435vw + 8.38843px) calc(.34435vw + 13.38843px)}.kep-login-facebook.metro{border-radius:0}.kep-login-facebook .fa{margin-right:calc(.34435vw + 3.38843px)}",
            "",
          ],
        ],
      }
    }
  />
</span>
`;
